ccloud-monitoring/README.md

```diff
@@ -5,3 +5,6 @@
 # Documentation
 
 You can find the documentation and instructions for running this Monitoring Apache Kafka Clients to Confluent Cloud example at [https://docs.confluent.io/platform/current/tutorials/examples/ccloud-monitoring/docs/index.html](https://docs.confluent.io/platform/current/tutorials/examples/ccloud-monitoring/docs/index.html?utm_source=github&utm_medium=example&utm_campaign=tm.devx_ch.examples_content.ccloud-monitoring#ccloud-monitoring-index).
+
+## Client Image
+The client docker image is stored in docker hub. If dependencies change for the client (ie the pom.xml changes), the client docker image will need to be recreated locally and pushed to docker hub. You can do this by running `make push-image`. 
```

Comment:
Public users won't have access to push to cnfldemos, so you might want to clarify who will be allowed to run this command.  If this is meant for public users to be able to do, then we probably need to clarify in the docs somewhere how to build and override the local usage of the image.

Reply:
Great point. I removed that note and put it our[ internal wiki page about "releases" ](https://confluentinc.atlassian.net/wiki/spaces/DEVX/pages/1809390195/How+To+Prep+Our+Repos+For+a+CP+Release#examples).